src/krkn_lib/models/telemetry/models.py

```diff
@@ -426,6 +457,7 @@ def __init__(self, json_dict: any = None):
             self.network_plugins = json_dict.get("network_plugins")
             self.run_uuid = json_dict.get("run_uuid")
             self.timestamp = json_dict.get("timestamp")
+            self.slo_alert = [SloAlert(a) for a in json_dict.get("slo_alert")]
```

Comment:
are you sure that the SloAlert constructor accepts a dictionary? for what I see it accept three params. I'm not sure that python is able to automatically map a dictionary and even if it's able would be better that you explicitly set the value of each of the three params(readability) OR you set a special named parameter calle json_dict=nil and you perform the deserialization in the constructor (See all the other data models)